data-serving/data-service/src/util/validate-env.ts

```diff
@@ -1,7 +1,16 @@
-import { cleanEnv, str, port } from 'envalid';
+import { CleanEnv, cleanEnv, str, port } from 'envalid';
 
-export default function validateEnv(): void {
-    cleanEnv(process.env, {
+export default function validateEnv(): Readonly<{
+    DB_CONNECTION_STRING: string;
+    DB_NAME: string;
+    DB_COLLECTION: string;
+    PORT: number;
+}> &
+    CleanEnv & {
+        readonly [varName: string]: string | undefined;
+        // eslint-disable-next-line indent
+    } {
+    return cleanEnv(process.env, {
         DB_CONNECTION_STRING: str({
             desc: 'MongoDB URI provided to MongoClient.',
             devDefault: 'mongodb://localhost/',
```

Comment:
do we want the mongod port here? mongodb://localhost:27017

Reply:
It's an optional url part, [and defaults to 27017](https://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html#basic-parts-of-the-url). Doesn't hurt to specify, though -- I'll add.